gql/lib/src/language/lexer.dart

```diff
@@ -378,10 +378,10 @@ class _Scanner {
           case 116: // \t
             break;
           case 117: // \u
-            final isUnicode = isHex(peek(offset: 1)) &&
-                isHex(peek(offset: 2)) &&
-                isHex(peek(offset: 3)) &&
-                isHex(peek(offset: 4));
+            final isUnicode = isHex(peek(offset: 1)!) &&
```

Comment:
It's actually a bug to use `!` operator here. (And it has been a bug before as well)

https://github.com/gql-dart/gql/pull/181/files#diff-6864373280eabf861af843f08282de1328566600ca3df35cd073908df9745f6fR381